internal/command/deploy/machines.go

```diff
@@ -53,6 +53,7 @@ type MachineDeploymentArgs struct {
 	UpdateOnly            bool
 	Files                 []*api.File
 	ProvisionExtensions   bool
+	NoExtensions          bool
```

Comment:
it needs a corresponding entry in: https://github.com/superfly/flyctl/blob/870516e939379bf565e51fb2338f1eee96b44cff/internal/command/deploy/machines.go#L147-L170

otherwise the value of --no-extensions is ignored